app/scripts/views/ready.js

```diff
@@ -97,7 +119,7 @@ function (Cocktail, FormView, Template, Url, Constants, ServiceMixin,
         type: this.type
       };
 
-      // spring campaign scheduled to launch 6/2
+      var marketingSnippet;
       if (this._able.choose('springCampaign2015')) {
         marketingSnippet = new MarketingSnippetiOS(marketingSnippetOpts);
```

Comment:
side-note: I guess we will need to update this very soon, whenever iOS gets shipped in U.S. :us: